docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,93 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+This section explains the process of creating a bootable Pop!\_OS USB in Linux using Popsicle.
+
+- [Install ISO flashing software.](/Getting-Started/Create-bootable-media/bootable-usb-using-linux.html#install-popsicle)
+- [Download the Pop!_OS.iso file.](/Getting-Started/Create-bootable-media/bootable-usb-using-linux.html#download-the-pop_osiso-file)
+- [Flash the ISO to your USB storage device.](/Getting-Started/Create-bootable-media/bootable-usb-using-linux.html#create-the-bootable-usb)
+
+---
+## Install Popsicle
+
+Popsicle is a Linux utility for flashing multiple USB devices at once. You can install Popsicle as an AppImage or a Flatpak.
+### AppImage
+
+1. Download the Popsicle AppImage from this [Popsicle repository.](https://github.com/pop-os/popsicle/releases/latest)
+
+2. Navigate to the downloaded AppImage. Right-click the AppImage and select `Properties`.
+
+3. Select the `Permissions` tab and check the `Execute` box.
+
+![AppImage Execute Permission](/images/create-bootable-usb-linux/appimage-execute.png)
+
+4. Double click the Popsicle AppImage to launch it.
+### Flatpak
+
+1. Follow the instructions on [Flatpak.org](https://flatpak.org/setup/) if Flatpak is not installed in your Linux distribution.
+
+2. Open a Terminal session and enter this command: 
+
+```bash
+flatpak install flathub com.system76.Popsicle
+```
+
+3. Enter this command to launch Popsicle: 
+
+```bash 
+flatpak run com.system76.Popsicle
+```
+## Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](/Getting-Started/Create-bootable-media/create-bootable-usb.html#choose-a-pop_os-image) for information about each version. 
+
+Take note of the hash value for your chosen ISO. Use this value to verify the integrity of the ISO file in step four of the installation process.
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
```

Comment:
Good placeholders, but I would have made the three placeholders not entirely identical since it is certain they will be different on the actual page.